test/integration_tests/python/test_containers.py

```diff
@@ -85,3 +85,44 @@ def test_switching_acquisition_between_projects(with_two_groups, data_builder, a
     data_builder.delete_session(session_1_id)
     data_builder.delete_session(session_2_id)
     data_builder.delete_project(project_id)
+
+
+def test_project_template(with_hierarchy, data_builder, as_user):
+    data = with_hierarchy
+
+    # create template for the project
+    r = as_user.post('/projects/' + data.project + '/template', json={
+        'session': { 'subject': { 'code' : '^testing' } },
+        'acquisitions': [{ 'label': '_testing$', 'minimum': 2 }]
+    })
+    assert r.ok
+    assert r.json()['modified'] == 1
+
+    # test non-compliant session (wrong subject.code and #acquisitions)
+    r = as_user.get('/sessions/' + data.session)
+    assert r.ok
+    assert r.json()['project_has_template'] is True
+    assert r.json()['satisfies_template'] is False
```

Comment:
Ah, found the [PEP recommendation](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) (At the very end of the section, CTRL+F "if greeting"):

> 
> Don't compare boolean values to True or False using == .
> 
> Yes:   if greeting:
> No:    if greeting == True:
> Worse: if greeting is True:
> 

For now I will just change from 'is' to '==' to make the usage consistent within the project.
Nice catch @kofalt, thanks!